sqlalchemy_redshift/dialect.py

```diff
@@ -301,6 +307,11 @@ def _fetch_redshift_column_attributes(self, column):
         return text
 
 
+class RedshiftExecutionCntext(PGExecutionContext_psycopg2):
+    def should_autocommit_text(self, statement):
```

Comment:
Yeah, that's better... I don't think that the psycopg2 dialect might need to add keywords very often, but it's the cleaner way

I'll see to do that later (before tomorrow? I've been away in the weekend, and now I'm a bit busy again)